File: Java/백준/Bronze/20492. 세금/세금.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int N = sc.nextInt();
        
        int case1 = (N / 100) * 78;
        int case2 = (N / 1000) * 956;
        
        System.out.println(case1 + " " + case2);
        
        sc.close();
    }
}

File: C++17/백준/Bronze/20492. 세금/세금.cc
#include <iostream>
using namespace std;

int main(void) {
    long long N;
    cin >> N;


    long long case1 = (N * 78) / 100;
    long long case2 = (N * 956) / 1000;

    cout << case1 << " " << case2 << endl;

    return 0;
}